lib/v2/bilibili/userChannelCollections.js

```diff
@@ -0,0 +1,54 @@
+const got = require('@/utils/got');
+const cache = require('./cache');
+const utils = require('./utils');
+
+const notFoundData = {
+    title: '此 bilibili 频道不存在',
+};
+
+module.exports = async (ctx) => {
+    const uid = Number.parseInt(ctx.params.uid);
+    const sid = Number.parseInt(ctx.params.sid);
+    const disableEmbed = ctx.params.disableEmbed;
+
+    const link = `https://space.bilibili.com/${uid}/channel/collectiondetail?sid=${sid}`;
+    const userName = await cache.getUsernameFromUID(ctx, uid);
+    const host = `https://api.bilibili.com/x/polymer/space/seasons_archives_list?mid=${uid}&season_id=${sid}&sort_reverse=false&page_num=1&page_size=10`;
+
+    const response = await got({
+        method: 'get',
+        url: host,
+        headers: {
+            Referer: link,
+        },
+    });
+
+    const data = response.data.data;
+    if (!data.archives) {
+        ctx.state.data = notFoundData;
+        return;
+    }
+
+    const channelName = data.meta.name;
+
+    ctx.state.data = {
+        title: `${userName} 的 bilibili 频道的合集 ${channelName}`,
+        link,
+        description: channelName,
+
+        item: data.archives.map((item) => {
+            const descList = [];
+            if (!disableEmbed) {
+                descList.push(utils.iframe(item.aid));
+            }
+            descList.push(`<img src="${item.pic}">`);
+            return {
+                title: item.title,
+                description: descList.join('<br>'),
+                pubDate: new Date(item.pubdate * 1000),
```

Comment:
```suggestion
                pubDate: parseDate(item.pubdate, 'X'),
```
docs: https://docs.rsshub.app/joinus/pub-date.html